### ISSUE TITLE
Unescaped left brace in regex is deprecated

### ISSUE THREAD

**Issue by eserte** [jhthorsen/mojolicious-plugin-assetpack#42]
New warnings with perl 5.21.x while running the test suite:
```
Unescaped left brace in regex is deprecated, passed through in regex; marked by <-- HERE in m/^html:before{ <-- HERE .*content:".*?\Wdummy.foo: No preprocessor defined for .*dummy\.foo";}/ at t/missing.t line 42.
# with-error files are always generated
Unescaped left brace in regex is deprecated, passed through in regex; marked by <-- HERE in m/^html:before{ <-- HERE .*content:"invalid\.foo: No preprocessor defined for .*dummy\.foo";}/ at t/missing.t line 42.
# with-error files are always generated
t/missing.t ........................ ok
Unescaped left brace in regex is deprecated, passed through in regex; marked by <-- HERE in m/body{ <-- HERE color:\#aaa}body{color:\#aaa}/ at t/my-plugin.t line 31, <DATA> line 2231.
Unescaped left brace in regex is deprecated, passed through in regex; marked by <-- HERE in m/body{color:\#aaa}body{ <-- HERE color:\#aaa}/ at t/my-plugin.t line 31, <DATA> line 2231.
# public: 
# t/read-only-with-source-assets t/read-only-with-existing-assets /var/tmp/cpansmoker-1001/mojo-assetpack-public/packed/..
t/my-plugin.t ...................... ok
```

**Comment by jhthorsen** [comment]
Thanks! I hope this is fixed in 8b0f308.

**Comment by eserte** [comment]
Yes, looks better with 0.42.
